src/Libraries/Microsoft.Extensions.AmbientMetadata.Application/README.md

```diff
@@ -1 +1,67 @@
-README
+# Microsoft.Extensions.AmbientMetadata.Application
+
+This provides runtime information for application-level ambient metadata such as the version, deployment ring, environment, and name.
```

Comment:
Should we motivate this a bit? The design intent is that this supplies state that is used by telemetry enrichers down the line.

Reply:
> Should we motivate this a bit?

@geeknoid can you explain what you mean by this?

Reply:
I mean this is kind of a strange component, I know David asked about it multiple times.

This component exists to capture data from the environment and populate it in DI in a convenient form where it can be consumed by enrichers. The design makes it possible to easily manually supply data to enable effective testing.

Reply:
I see, so are you saying we don't expect folks to depend on this directly, and instead depend on some of the Diagnostic or Telemetry packages that provide enrichment and for this to be a transitive dependency?

Reply:
@tekian How is this used in practice by our customers? Do they depend on it explicitly, or does it come in for free? I think customers still need to explicitly initialize the component themselves, rigth?

Reply:
```suggestion
This flows runtime information for application-level ambient metadata such as the version, deployment ring, environment, and name. This information can be useful to enrich telemetry and logs.
```